deploy/README.md

```diff
@@ -1,3 +0,0 @@
-# 301 Moved
-
-This document is now moved to [Deployment Guides](../docs/deployment).
```

Comment:
This file was kept around because the link to the file has been shared a lot in the past and there will be users who would land up here. Rather than giving them a bad Github 404, we want the users to have a smoother experience navigating our docs.

Can we not remove this?

Reply:
@UkiahSmith can you please follow-up?

Reply:
Yes, we can keep the file and update the main Readme's link.

Reply:
I've updated this and pushed it.